src/image/pixels.js

```diff
@@ -636,74 +657,86 @@ function parseFilterArgs(...args) {
 }
 
 /**
- * Get a region of pixels, or a single pixel, from the canvas.
- *
- * Returns an array of [R,G,B,A] values for any pixel or grabs a section of
- * an image. If no parameters are specified, the entire image is returned.
- * Use the x and y parameters to get the value of one pixel. Get a section of
- * the display window by specifying additional w and h parameters. When
- * getting an image, the x and y parameters define the coordinates for the
- * upper-left corner of the image, regardless of the current <a href="#/p5/imageMode">imageMode()</a>.
- *
- * Getting the color of a single pixel with get(x, y) is easy, but not as fast
- * as grabbing the data directly from <a href="#/p5/pixels">pixels[]</a>. The equivalent statement to
- * get(x, y) using <a href="#/p5/pixels">pixels[]</a> with pixel density d is
- * ```javascript
- * let x, y, d; // set these to the coordinates
- * let off = (y * width + x) * d * 4;
- * let components = [
- *   pixels[off],
- *   pixels[off + 1],
- *   pixels[off + 2],
- *   pixels[off + 3]
- * ];
- * print(components);
- * ```
- * See the reference for <a href="#/p5/pixels">pixels[]</a> for more information.
- *
- * If you want to extract an array of colors or a subimage from an p5.Image object,
- * take a look at <a href="#/p5.Image/get">p5.Image.get()</a>
+ * Gets a pixel or a region of pixels from the canvas. `get()` is easy to use
+ * but it's not as fast as <a href="#/p5/pixels">pixels</a>.
+ *
+ * The version of `get()` with no parameters returns the entire canvas.
+ *
+ * The version of `get()` with two parameters interprets them as
+ * coordinates. It returns an array with the `[R, G, B, A]` values of the
+ * pixel at the given point.
+ *
+ * The version of `get()` with four parameters interprets them as coordinates
+ * and dimensions. The first two parameters are the coordinates for the
+ * upper-left corner of the image, regardless of the current
+ * <a href="#/p5/imageMode">imageMode()</a>. The last two parameters are the
+ * width and height of the image section.
+ *
+ * Use <a href="#/p5.Image/get">p5.Image.get()</a> to work directly with
```

Comment:
This method is also available on `p5.Graphics`, but I'm not sure we have separate docs for that or mention it anywhere. Is it worth mentioning that here too, if we're already mentioning how something similar exists on `p5.Image`?

Reply:
Ah, yeah good point. Probably not something to address in this issue then, but for longer term changes, what do you feel would be the best way to manage this sort of thing? Some docs pages mention when classes extend other classes and have a link to the super class docs page under the list of methods, but class hierarchy might be kind of confusing to newcomers.  We could maybe list every method, including inherited ones? (Ideally in an automated way to avoid copy-and-pasting docs, where copies could go out of date?)

Reply:
Oops, I just reread the source and found where `p5.Graphics` [binds methods](https://github.com/processing/p5.js/blob/1c2273d6c9113b8ae279de8f0921fabebbc87095/src/core/p5.Graphics.js#L45) from the `p5` instance that's passed during construction. Now I get that `p5.Graphics` objects arise from a flavor of multiple inheritance with `p5` and `p5.Element`. That's way too complex for newcomers.

I like your idea to automatically include documentation from super classes. Agreed it's helpful for completeness and maintainability with subclasses. For `p5.Graphics` specifically, I think it's worth emphasizing and revising that "The fields and methods for this class are extensive, but mirror the normal drawing API for p5."